packages/dx-react-chart/src/templates/series/area.jsx

```diff
@@ -27,7 +29,7 @@ class RawArea extends React.PureComponent {
 }
 
 RawArea.propTypes = {
-  path: PropTypes.func.isRequired,
+  path: PropTypes.func,
```

Comment:
The `seriesComponent` in area, line and spline series receive not required path that should be specified in the documentation.

Reply:
Done.